### PR TITLE
Get involved page v1.1

### LINKED ISSUE #206: Get Involved page v1.1 - Convert base layout for default view
Implement base layout of `Get Involved` page v1.1 for default view (Excluding invite process component)

>`Story points: 3`

### LINKED ISSUE #207: Get Involved page v1.1 - Convert invite component for default view
Implement invite component layout of `Get Involved` page v1.1 for default view
Integrate it with `Get Involved` page base layout

>`Story points: 3`

### LINKED ISSUE #208: Convert Get Involved page v1.1 responsiveness
Implement `Get Involved` page v1.1 responsiveness to fit all it's breakpoints (XL, L, M, S, XS, XXS and XXXS)

>`Story points: 4`

### LINKED ISSUE #248: Get Involved - FAQ text incorrect
The text exposed for the FAQs on Get Involved page is incorrect except for the last question. Tested as part of PR #236 

https://deploy-preview-236--keen-stonebraker-bb1a10.netlify.com/get-involved/#what-do-i-need-in-order-to-browse-the-safe-network?
https://deploy-preview-236--keen-stonebraker-bb1a10.netlify.com/get-involved/#do-i-need-an-account-to-access-the-network?
https://deploy-preview-236--keen-stonebraker-bb1a10.netlify.com/get-involved/#how-do-i-create-an-account?
https://deploy-preview-236--keen-stonebraker-bb1a10.netlify.com/get-involved/#how-do-i-get-involved-with-the-safe-network-community?
https://deploy-preview-236--keen-stonebraker-bb1a10.netlify.com/get-involved/#how-do-i-get-help?
CF, the final question is the correct text:
https://deploy-preview-236--keen-stonebraker-bb1a10.netlify.com/get-involved/#who-are-maidsafe?

### LINKED ISSUE #249: Get Involved - More FAQs button doesn't link
image

Tested on Netlify preview for PR #236 https://deploy-preview-236--keen-stonebraker-bb1a10.netlify.com/get-involved/#

More FAQs button doesn't link to the FAQ page as expected.

### LINKED ISSUE #250: Get Involved - claim an invite
Testing PR #236 - Netlify link: https://deploy-preview-236--keen-stonebraker-bb1a10.netlify.com/get-involved/#

There is a typo in the claim an invite section:
"You’ll need to _claim an one_ to get started."
image

### LINKED ISSUE #252: Get Involved page not showing all elements at Small size
**Steps to reproduce:**
1. Launch safenetwork.tech - tested on Netlify preview: https://deploy-preview-236--keen-stonebraker-bb1a10.netlify.com/get-involved/
2. Resize browser to less than 1300 pixels wide (design Small size).
3. Scroll to "Be part of the community" section.

**Expected:**
image
**Actual:**
image
(Element of background image of water is not displayed.)

### PR DESCRIPTION
**Pull request by shankar2105**
Fixes #206 Convert Get involved page default view
Fixes #207 Implement Invite component
Fixes #208 Implement responsiveness

---
**Resolved issues**
Fixes #248 - Update FAQ content
Fixes #249 - Update More FAQs link
Fixes #250 - Typo issue in `Claim an Invite` section
Fixes #252 - Z index issue in texture component

### CODE REVIEW

**Comment by JimCollinson** [comment]
`overline` text should have `letter-spacing: 0.2em` and opacity of `0.5`

**Comment by JimCollinson** [comment]
Modal Issues, XS and smaller:

- Modal should take over the whole screen (full height white background) with main content vertically centered.
- Fixed header with drop shadow for the `Get An Invite` header and close button. Rest should be scrollable.
- Content behind should not be scrollable

**Comment by JimCollinson** [comment]
Still need to mock up the `Why do I need an invite` content, which may mean scrolling content in the modal... so content behind scroll may not be ideal.

**Comment by JimCollinson** [comment]
Is there is any way to stop the dropshadow showing at the top of the `Be part of the community` container, as per the Figma file, that would be great!

**Comment by JimCollinson** [comment]
`AccordionItem__title` `H3` should have an opacity of 0.7

**Comment by JimCollinson** [comment]
The Dev Hub button should be the 'subtle' style, which means it's background has `0.5` opacity

**Comment by JimCollinson** [comment]
At `XXS` `H3`s should have `margin-bottom: 24px`

**Comment by JimCollinson** [comment]
For `.underlinedLink:after` remove `margin-top`. 

`padding-right` should also be `1px`

**Comment by JimCollinson** [comment]
The H3 in the `Get Started` panel (e.g. Claim an Invite) is acting like an accordion item: it's cursor changes to pointer, and it has the wrong opacity etc.

Should just be a normal `H3`

**Comment by JimCollinson** [comment]
The External link buttons aren't quite right (wrong padding/spacing around the arrow for example) but let's fix these in a separate PR where we can sort out all the button styles and states etc.

**Comment by JimCollinson** [comment]
Font size for the `Get an Invite` and `Download for X` buttons should be 20px. 

(This is different from the normal button, and is an exception for these particular buttons on this page, so doesn't need to be a global change)

**Comment by JimCollinson** [comment]
This is the only thing I can see missing:

> overline text should have ~letter-spacing: 0.2em and~ opacity of 0.5

File: src/v1.1/pages/GetInvolved/Invite/Invite.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  }
+
+  toggleInviteModal(state = false) {
+    lockBodyScroll(state)
+    this.setState({
+      showInviteOpts: state
+    })
+  }
+
+  detectPlatform() {
```
Comment (joshuef) [inline src/v1.1/pages/GetInvolved/Invite/Invite.js]: this seems like a `util`

**Pull request merged**